package.json

```diff
@@ -55,20 +60,20 @@
     "@playwright/test": "1.31.1",
     "@types/jasmine": "4.3.1",
     "@types/node": "18.13.0",
-    "@typescript-eslint/eslint-plugin": "^5.43.0",
-    "@typescript-eslint/parser": "^5.43.0",
+    "@typescript-eslint/eslint-plugin": "5.56.0",
+    "@typescript-eslint/parser": "5.56.0",
     "@wdio/cli": "8.3.8",
     "@wdio/jasmine-framework": "8.3.8",
     "@wdio/local-runner": "8.3.8",
     "@wdio/spec-reporter": "8.3.0",
-    "chromedriver": "111.0.0",
-    "eslint": "^8.28.0",
+    "chromedriver": "110.0.0",
```

Comment:
Why do we need to revert to 110.0.0? Can we use the latest version?

Reply:
chromedriver updated to 117.0